dao/SettledBalanceDao.java

```diff
@@ -0,0 +1,202 @@
+package dao;
+
+import java.io.*;
+import java.util.List;
+import java.util.ArrayList;
+import java.math.BigDecimal;
+
+import model.data.Balance;
+/**
+  *銘柄の確定残高ファイルに関する処理をするクラス。
+  */
+public class SettledBalanceDao {
+  /** 銘柄の確定残高ファイルパス */
+  private final String filePath = "csv/settledbalancedata.csv";
+  /**
+    *銘柄のコードの保有数量と簿価を取り出すメソッド
+    * @param code　探索する銘柄コード
+    * @return 探索コードと等しい銘柄情報の配列
+    */
+  public Balance getBalanceData(String code) {
+    Balance balance = null;
+    BufferedReader br = null;
+    try {
+      br = new BufferedReader(new FileReader(this.filePath));
+
+      String line = null;
+      while((line = br.readLine()) != null) {
+        String[] bondData = line.split(",",-1);
```

Comment:
```suggestion
        String[] settledBalanceData = line.split(",", -1);
```
`bondData`という名前だと銘柄マスタファイルのデータだと解釈されかねない（実際、L31を読むまでマスタデータだと思って読んでいた、、）

Reply:
```suggestion
        String[] settledBalanceData = line.split(",", -1);
```
bondDataだとマスタファイルのデータだと誤解されかねない気がします

Reply:
すいません。修正しました！